README.md

```diff
@@ -1,30 +1,26 @@
-<img src="docs/images/notary-blk.svg" alt="Notary" width="400px"/>
-
 [![GoDoc](https://godoc.org/github.com/theupdateframework/notary?status.svg)](https://godoc.org/github.com/theupdateframework/notary)
 [![Circle CI](https://circleci.com/gh/theupdateframework/notary/tree/master.svg?style=shield)](https://circleci.com/gh/theupdateframework/notary/tree/master) [![CodeCov](https://codecov.io/github/theupdateframework/notary/coverage.svg?branch=master)](https://codecov.io/github/theupdateframework/notary) [![GoReportCard](https://goreportcard.com/badge/theupdateframework/notary)](https://goreportcard.com/report/github.com/theupdateframework/notary)
 [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Ftheupdateframework%2Fnotary.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2Ftheupdateframework%2Fnotary?ref=badge_shield)
 
 # Notice
 
-The Notary project has officially been accepted in to the Cloud Native Computing Foundation (CNCF).
-It has moved to https://github.com/theupdateframework/notary. Any downstream consumers should update
-their Go imports to use this new location, which will be the canonical location going forward.
-
-We have moved the repo in GitHub, which will allow existing importers to continue using the old
-location via GitHub's redirect.
+This repository serves as an implementaiton of 
+*[The Update Framework specficaitions](https://github.com/theupdateframework/specification)* 
+and all references to "Notary" in this repository refer to the implementation of the client 
+and server aligning with the [TUF](https://github.com/theupdateframework/specification) specification.
 
 # Overview
 
-The Notary project comprises a [server](cmd/notary-server) and a [client](cmd/notary) for running and interacting
+This repository comprises of a [server](cmd/notary-server) and a [client](cmd/notary) for running and interacting
 with trusted collections. See the [service architecture](docs/service_architecture.md) documentation
 for more information.
 
-Notary aims to make the internet more secure by making it easy for people to
+The aim is to make the internet more secure by making it easy for people to
 publish and verify content. We often rely on TLS to secure our communications
 with a web server, which is inherently flawed, as any compromise of the server
 enables malicious content to be substituted for the legitimate content.
 
-With Notary, publishers can sign their content offline using keys kept highly
+Publishers can sign their content offline using keys kept highly
 secure. Once the publisher is ready to make the content available, they can
 push their signed trusted collection to a Notary Server.
```

Comment:
Should we change "Notary Server" to "`notary` server component"

Reply:
Update to just `notary server`. Felt component was quite verbose.